packages/node_modules/@webex/react-container-message-composer/src/container.js

```diff
@@ -177,8 +180,21 @@ export class MessageComposer extends Component {
       } = props;
 
       const files = constructFiles(e.target.files);
-
-      props.addFiles(conversation, activity, files, sparkInstance);
+      const overThreshold = files.some((file) => file.overThreshold);
+
+      if (overThreshold) {
+        props.addError({
+          id: 'file-attachment-threshold',
+          actionTitle: 'Dismiss',
+          onAction: () => props.removeError('file-attachment-threshold'),
+          displayTitle: 'What if I told you',
+          displaySubtitle: 'A file you are attempting to attach is too large and will crash the browser',
```

Comment:
I think we should let the user know exactly what they did wrong
```suggestion
          displaySubtitle: Only attachments smaller than 150MB are allowed' ,
```